docs/cog_guides/general.rst

```diff
@@ -0,0 +1,214 @@
+.. _general:
+
+=======
+General
+=======
+
+This is the cog guide for the general cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load general
+
+.. _general-usage:
+
+-----
+Usage
+-----
+
+This cog includes a miscellaneous group of games, useful
+tools, and informative commands such as ``serverinfo`` or ``urban``.
+
+.. _general-commands:
+
+--------
+Commands
+--------
+
+Here's a list of all commands available for this cog.
+
+.. _general-command-8:
+
+^^^^^^^^^
+8 (8ball)
+^^^^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]8 <question>
+
+**Description**
+
+Ask 8 ball a question. 
+
+.. note:: Your question must end with a question mark.
+
+**Arguments**
+
+* ``<question>``: The question you would like to ask 8 ball.
+
+.. _general-command-choose:
+
+^^^^^^
+choose
+^^^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]choose [choices...]
+
+**Description**
+
+Choose between multiple options.
+
+.. note:: To denote options which include whitespace, you should use
+    double quotes.
+
+**Arguments**
+
+* ``<choices>``: The arguments for Red to randomly choose from.
+
+.. _general-command-flip:
+
+^^^^
+flip
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]flip [user]
+
+**Description**
+
+Flip a coin... or a user.
+
+**Arguments**
+
+* ``<user>``: The user to flip. Defaults to None.
```

Comment:
Maybe instead of "defaults to None" (which means nothing to an end user), say "defaults to flipping a coin" / "flips a  coin if no user is passed" or something like that?